crates/policy/src/lib.rs

```diff
@@ -67,6 +70,9 @@ impl PolicyFactory {
         let mut config = Config::default();
         config.async_support(true);
         config.cranelift_opt_level(wasmtime::OptLevel::Speed);
+        config
+            .cache_config_load_default()
+            .map_err(LoadError::CacheSetup)?;
```

Comment:
This loads the default cache config, which should help with speeding up the tests.

On my laptop, without cache it looks like it adds ~1s to each test